first-android-submission.md

```diff
@@ -0,0 +1,53 @@
+# First Submission of an Android App
+
+#### 🤔 What Happened
+
+If you're submitting your Android app to Google Play Store for the first time you need to do it manually. If you tried to take advantage of the `expo upload:android` you most probably got the following error:
+
+> You haven't submitted this app to Google Play Store yet. The first submission of the app needs to be performed manually.
```

Comment:
```suggestion
```

similar to other comment, this makes it easier for us to improve error messages in cli without needing to worry about updating expo.fyi